test/hack/checkLabels/main.go

```diff
@@ -44,7 +44,7 @@ func getLabels(ruleList []promv1.PrometheusRule, matcher string) []string {
 	for _, p := range ruleList {
```

Comment:
Can this be moved into a function? GetAllPrometheusRulesLabels or smth? nesting is ewww, especially 4 levels of them :D

Reply:
Me and my poor coding skills are the ones to blame for this !

Reply:
I'm not blaming anyone <3

Reply:
Goosebumps might have been to much :) But let's pair next time :)

Reply:
No worries ! Sure I'd love to :)

Reply:
I get you Quentin, but for such a small script I would leave this as is for now.

Reply:
Well, I would tend to think that that's what the opsctl creator thought at the time :D but ok 👍🏻